prismarine-viewer/viewer/lib/entities.ts

```diff
@@ -833,3 +842,26 @@ function removeArmorModel (entityMesh: THREE.Object3D, slotType: string) {
     }
   }
 }
+
+function addMapModel(entityMesh: THREE.Object3D, mapNumber: number) {
+  const imageData = bot.mapDownloader.maps?.[mapNumber] as any as string
+  if (!imageData) return
+
+  const material = new THREE.MeshLambertMaterial({
+    transparent: true,
+    alphaTest: 0.1
+  })
+  loadTexture(imageData, texture => {
```

Comment:
will texture be removed when the map entity is removed? from the current code it doesn't seem so, would be great if you could test:
- in console type: `viewer.renderer.info.memory.textures` - this number should not increase after removing / re-adding, thanks